plugins/inputs/monit/status_response_0.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Move all test files to a subdirectory called `testdata`.  This is the idiomatic directory for testing files and has special handling in `go test`.  From the output of `go help test`:

> The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.

Reply:
Done. Please check.